src/pipe/graph.c

```diff
@@ -368,6 +368,8 @@ VkResult dt_graph_run(
       return VK_INCOMPLETE;
     }
 
+    QVKR(vkResetCommandPool(qvk.device, graph->command_pool, 0));
```

Comment:
okay maybe your validation layers are newer than mine or i didn't enable the best practices notices there.. anyways, if you want to reset the whole pool, we'll need one pool per command buffer. the graph has two for the odd/even pipelines when interleaving two subsequent frames on the gpu at the same time. especially amd seems to profit a lot from this scheme. in any case we'll need two command pools, one for each of the two command buffers, and then reset the one for the command buffer in question. probably there are some more places in the graph where commands are issued (uploading array images).

Reply:
my intention with this PR is not introduce a big change like this, I will revert all the command pool changes and leave only the sampler bindings warning fixes, I can get back to to this in another moment.